<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>138</width>
         <height>590</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_Open">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>141</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_NextImage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>141</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Next Image</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_PrevImage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>141</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>Prev Image</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Save">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>250</y>
          <width>141</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Create">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>340</y>
          <width>141</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>Create Pixmap</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Dlete">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>430</y>
          <width>141</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete Line</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>3000</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>578</width>
         <height>590</height>
        </rect>
       </property>
       <zorder>scrollArea_3</zorder>
       <zorder>scrollArea_3</zorder>
       <zorder>scrollArea_3</zorder>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>198</width>
         <height>590</height>
        </rect>
       </property>
       <widget class="QToolButton" name="toolButton_Config">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>目标配置</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_ColorChoose">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>目标种类选择：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_ColorChoose">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>人：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>轿车、面包车：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>卡车：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>320</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>客车、公交车：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>360</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>自行车：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>400</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>摩托车：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>440</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>电瓶车：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>480</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>三轮车：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category1_color">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>200</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category2_color">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>240</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category3_color">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>280</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category4_color">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>320</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category5_color">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>360</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category6_color">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>400</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category7_color">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>440</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Category8_color">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>480</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuThsfh">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCreate_Pixmap"/>
    <addaction name="actionFill_Color"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuThsfh"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_Dir">
   <property name="text">
    <string>Open Dir</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCreate_Pixmap">
   <property name="text">
    <string>Create Pixmap</string>
   </property>
  </action>
  <action name="actionEdit_Label">
   <property name="text">
    <string>Edit Label</string>
   </property>
  </action>
  <action name="actionFill_Color">
   <property name="text">
    <string>Delete Line</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
